src/tokens/AssetList.test.tsx

```diff
@@ -259,42 +254,14 @@ describe('AssetList', () => {
   })
 
   it.each([
-    {
-      name: 'tokens tab on wallet tab when gate is off',
```

Comment:
is this test removed, I don't see it below but it looks like it is for tab navigator?

Reply:
this test is for when the feature gate is off, we're rolling out the on case, so this specific case isn't needed.

Reply:
ahh okay I was confused because isWalletTab is set to true but then gate is off

Reply:
good catch, was confused because there are different gates involved, will add it back